code/datums/statistics/stat_helpers.dm

```diff
@@ -155,15 +155,10 @@
 	station_name = station_name()
 
 	// check for survivors
-	for(var/datum/mind/M in ticker.minds)
-		// i'd like manifest stats, eventually, to be something that doesn't depend on
-		// all data being 'pristine' at round end; for instance, an ayy'd player
-		// will probably have a different entry than what they were at roundstart
-		// and that's gross
-		manifest_entries.Add(new /datum/stat/manifest_entry(M))
-		if(istype(M.current, /mob/living) && !M.current.isDead())
-			add_survivor_stat(M.current)
-
+	for(var/mob/living/M in player_list)
+		manifest_entries.Add(new /datum/stat/manifest_entry(M.mind))
+		if(!M.isDead())
+			add_survivor_stat(M)
```

Comment:
This isn't the same thing as `for(var/datum/mind/M in ticker.minds)`. With this change, anyone who has logged out (thus being removed from player_list, which only tracks mobs with clients) will not show up on the scoreboard.

I recommend you change this to `mob_list` and add a check for a mind. At the same time, this kind of problem demonstrates why a global list of minds is a good thing, even if `ticker.minds` is poorly implemented/utilized.